help/include/global-feed.md

```diff
@@ -13,8 +13,8 @@ view](/help/configure-home-view#configure-home-view) for the Zulip web app.
 
 {tab|mobile}
 
-1. Tap the **All messages**
-   (<img src="/static/images/help/mobile-globe-icon.svg" alt="globe" class="help-center-icon"/>)
+1. Tap the **Global feed**
+   (<img src="/static/images/help/mobile-globe-icon.svg" alt="globe" class="mobile-icon"/>)
```

Comment:
What's the story with the icon calss name change? Is `mobile-icon` a real class?